docs/devel/issues.md

```diff
@@ -33,23 +33,21 @@ Documentation for other releases can be found at
 GitHub Issues for the Kubernetes Project
 ========================================
 
-A list quick overview of how we will review and prioritize incoming issues at https://github.com/kubernetes/kubernetes/issues
+A quick overview of how we will review and prioritize incoming issues at https://github.com/kubernetes/kubernetes/issues
 
 Priorities
 ----------
 
-We will use GitHub issue labels for prioritization.  The absence of a priority label means the bug has not been reviewed and prioritized yet.
+We use GitHub issue labels for prioritization.  The absence of a
+priority label means the bug has not been reviewed and prioritized
```

Comment:
I'd like to keep the description shorter and focus on priority, not general review.  I'll remove the word reviewed to avoid the confusion you mention.